concordia/templates/transcriptions/asset_detail.html

```diff
@@ -146,17 +146,19 @@
                     </span>
                 </div>
 
-                <form id="transcription-editor" class="ajax-submission flex-grow-1 d-flex flex-column" method="post" action="{% url 'save-transcription' asset_pk=asset.pk %}" data-transcription-status="{{ transcription_status }}" {% if transcription %}data-transcription-id="{{ transcription.pk|default:'' }}" {% if transcription.submitted %}data-unsaved-changes="true"{% endif %} data-submit-url="{% url 'submit-transcription' pk=transcription.pk %}" data-review-url="{% url 'review-transcription' pk=transcription.pk %}"{% endif %}>
+                <form id="transcription-editor" class="ajax-submission flex-grow-1 d-flex flex-column d-print-block" method="post" action="{% url 'save-transcription' asset_pk=asset.pk %}" data-transcription-status="{{ transcription_status }}" {% if transcription %}data-transcription-id="{{ transcription.pk|default:'' }}" {% if transcription.submitted %}data-unsaved-changes="true"{% endif %} data-submit-url="{% url 'submit-transcription' pk=transcription.pk %}" data-review-url="{% url 'review-transcription' pk=transcription.pk %}"{% endif %}>
                     {% csrf_token %}
                     <input type="hidden" name="supersedes" value="{{ transcription.pk|default:'' }}" />
 
                     <h2>Transcription</h2>
 
                     {% spaceless %}
-                        <textarea readonly class="form-control w-100 rounded flex-grow-1" name="text" id="transcription-input" placeholder="{% if transcription_status == 'not_started' or transcription_status == 'in_progress' %}Go ahead, start typing. You got this!{% else %}Nothing to transcribe{% endif %}" aria-label="Transcription input">
+                        <textarea readonly class="form-control w-100 rounded flex-grow-1 d-print-none" name="text" id="transcription-input" placeholder="{% if transcription_status == 'not_started' or transcription_status == 'in_progress' %}Go ahead, start typing. You got this!{% else %}Nothing to transcribe{% endif %}" aria-label="Transcription input">
                             {{ transcription.text }}
                         </textarea>
 
+                        <div class="print-transcription-text" aria-hidden="true" style="display: none;">{{ transcription.text }}</div>
```

Comment:
Following the pattern of the rest of the page, shouldn't we use the Bootstrap classes rather than an inline `style`?

```suggestion
                        <div class="print-transcription-text d-none d-print-block" aria-hidden="true">{{ transcription.text }}</div>
```

Reply:
Actually that was intentional to prevent it from displaying when CSS is disabled.

Reply:
When would we support disabling CSS and in what context does that not also disable inline styles?

Reply:
I meant an user turning off stylesheets from their browser. For testing, slow connection and etc reasons. You can test it with Firefox view > page style > no style. Probably only a few would have this setting, but wanted to be sure that we'll never display two identical text blocks in any circumstances.

Reply:
Slow connections will block the page rendering until the stylesheets have loaded